clean/ca/config/palm_springs_pd.py

```diff
@@ -0,0 +1,15 @@
+index_request_headers = {
+    "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7",
+    "accept-language": "en-US,en;q=0.9",
+    "cache-control": "max-age=0",
+    "priority": "u=0, i",
+    "sec-ch-ua": '"Not)A;Brand";v="99", "Google Chrome";v="127", "Chromium";v="127"',
+    "sec-ch-ua-mobile": "?0",
+    "sec-ch-ua-platform": '"Windows"',
+    "sec-fetch-dest": "document",
+    "sec-fetch-mode": "navigate",
+    "sec-fetch-site": "none",
+    "sec-fetch-user": "?1",
+    "upgrade-insecure-requests": "1",
+    "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36",
```

Comment:
It looks like [this is overwriting the default](https://github.com/biglocalnews/clean-scraper/pull/93/files#diff-9f4ed833caf9b56cd557acb6612eb183c232b5e4535094fd7da9e289811adcf9R63) `user-agent` set by the method. Does removing this header break the scraper? We want to maintain the user agent `Big Local News (biglocalnews.org)` to avoid getting blocked by their servers

Reply:
I have removed the user-agent and it still works, but in utils it is setting up an other Cap case `User-Agent`,
If we add the indent at line 388 then it will only add the user-agent where the headers are not passed, I think the better way of doing it is that when passing the `headers ` do-not pass the `user-agent` if possible and change the utils to set the default `user-agent` even if the headers are available and `user-agent` is not in headers dict.

Reply:
That sounds reasonable to me. Maybe since we already know the capitalization can be unpredictable, that section in utils can test for both and only include it if neither is set:
```
    if "headers" not in kwargs:
        kwargs["headers"] = {}
    if "user-agent" not in kwargs["headers"] and "User-Agent" not in kwargs["headers"]:
        kwargs["headers"]["User-Agent"] = user_agent
```

Reply:
@stucka yes I agree.